active/fellowship.md

```diff
@@ -2,30 +2,67 @@
+- Designating one member of the committee to act as the line manager for active fellows. See [Fellow Management](#fellow-management), below, for details of this role.
+- Acting as an HR committee for Fellows. In the event of a complaint about a Fellow, or if there are performance issues, the Fellowship-WG will investigate, in concert with the [Code of Conduct WG](code-of-conduct.md) if necessary, and present findings and a recommendation to the board for action.
+  - As with the CoC-WG, the Fellowing-WG has the power to unilaterally suspend Fellows, with pay, pending an investigation. This power should only be used in egregious circumstances, where waiting on investigation and/or board action might plausibly be harmful.
+- Acting as an escalation point if a Fellow has a disagreement with their designated manager (or that person isn't available, etc). The escalation path for Fellows is: designated manager -> full Fellowship WG -> Board.
+- Gathering and maintaining statistics on the impact and efficacy of the Fellow program (for use in fundraising, reports to sponsors, etc). See [reporting](#reporting) for a short discussion of this report.
+
+### Fellow Management
+
+One member of the committee is designated as the manager for the Fellows. It's expected that this person will have previous engineering management experience, and the Board must approve this designation.
+
+Defining the specifics of this role is out of scope for this document -- it should be developed and maintained by consensus of the Fellows and the WG members and laid out in the documentation mentioned above.
+
+Broadly speaking, this should be thought of as a "supportive" management role, rather than a "directive" one. The DSF intends to grant a very high degree of autonomy to Fellows. The DSF trusts their judgement on what's right to work on. So the person designated as the Fellows manager is there to provide the Fellows whatever support they need to be happy and productive. This may include help with setting priorities, coaching, and so forth -- subject to what each Fellow needs. It's suggested that the Fellows hold one-on-ones with their manager at least monthly.
+
+The designated manager is also the official approver for expenses for Fellows, subject to the [budget](#budget), see below.
 
 ## Initial membership
 
-- Chair: Brian Moloney
-- Board liaison: Chaim Kirby
+Upon adoption of this new charter, the Board will issue an open call for members, recruit, and select membership. We'll follow the guidelines outlined below under [future membership](#future-membership) in selecting this initial group./
+
+- Chair: TBD
+- Board liaison: TBD
+- Designated manager: TBD
 - Other members:
-  - Andrew Godwin
-  - Frank Wiles
+  - TBD
 
 ## Future membership
```

Comment:
Do we want the Fellows to have a say in the selection processes?

Reply:
I super hope you will :)

Reply:
Added a line to this effect to the charter to make it explicit because explicit is yada yada